packages/client/mutations/toasts/mapPromptToJoinOrgToToast.ts

```diff
@@ -0,0 +1,45 @@
+import graphql from 'babel-plugin-relay/macro'
+import {Snack} from '../../components/Snackbar'
+import {OnNextHistoryContext} from '../../types/relayMutations'
+import {mapPromptToJoinOrgToToast_notification$data} from '../../__generated__/mapPromptToJoinOrgToToast_notification.graphql'
+import SendClientSegmentEventMutation from '../SendClientSegmentEventMutation'
+import makeNotificationToastKey from './makeNotificationToastKey'
+
+graphql`
+  fragment mapPromptToJoinOrgToToast_notification on NotifyPromptToJoinOrg {
+    id
+    activeDomain
+  }
+`
+
+const mapPromptToJoinOrgToToast = (
+  notification: mapPromptToJoinOrgToToast_notification$data,
+  {atmosphere}: OnNextHistoryContext
+): Snack => {
+  const {id: notificationId, activeDomain} = notification
+
+  return {
+    autoDismiss: 0,
+    showDismissButton: true,
+    key: makeNotificationToastKey(notificationId),
+    message: `Your teammates at ${activeDomain} are loving Parabol! Do you want to join them?`,
+    onManualDismiss: () => {
+      SendClientSegmentEventMutation(atmosphere, 'Snackbar Clicked', {
```

Comment:
I added this because we have similar for some other snackbars. Every time you click anywhere the snackbar is dismissed, so I'm tracking clicks here. However tagging @tghanken to see what actual events do we need in this PR

Reply:
-1 In general, we want a way to track people through the funnel of actions on something like this. View -> Action -> Conversion.

For this specifically, I think we have two options:
1. Emit a `Snackbar Viewed` event with a `snackbarType: 'promptToJoinOrg'` parameter when we originally display the snackbar, followed by emitting this `Snackbar Clicked` event under the `action: {callback: ...}` section instead of under `onManualDismiss`.
2. Don't add the `Snackbar Viewed` event, and instead add `Snackbar Clicked` to the action callback with a `actionType: 'joinRequest'` parameter in addition to this existing `onManualDismiss` event.

Either way should let us drill into the conversion rate of the specific snackbar.  Of the two, I prefer option 1, because it's easier on our backend tooling, but we can make either work!